wurst/_handles/primitives/Integer.wurst

```diff
@@ -34,4 +34,8 @@ public function int.pow(int x) returns int
 
 /** Linear Interpolation with alphafactor(smoothness) */
 public function int.lerp(int target, real alpha) returns int
-	return ((this * (1-alpha)) + (target * alpha)).round()
\ No newline at end of file
+	return ((this * (1-alpha)) + (target * alpha)).round()
+
+/** Checks if this int is between low and high value */
+public function int.valueBetween(int low, int high) returns bool
```

Comment:
I don't particularly like this naming "valueBetween". Imo it refers more to generating a value between those numbers.
A check should probably wordes "isBetween" or "isInBounds" or something.